File: Easy-exercises/Argentina/Argentina.cpp
#include<iostream>
using namepace std;
#define CENT_RATE 2
int main(){
    int pesos, dollar, rate;
    cout <<"enter value for Pesos: " << endl;
    cin >> pesos;
    cout << "enter value for Dollar: " << endl;
    cin >> dollar;
   //convert dollar to pesos
    rate = (50 * dollar);
    rate > pesos ? cout << "Pesos";
                 : cout << "Dollars";
    return 0;
}

File: Easy-exercises/Argentina/Argentina.php
<?php
    echo ("enter value for Pesos: \n");
    $pesos = (int)readline('');
    echo ("enter value for Dollar: \n");
    $dollar = (int)readline('');
   //convert dollar to pesos
    $rate = (50 * $dollar);
    if($rate > $pesos)
    echo ("Pesos");
    else
    echo("Dollars");
?>